scipy/signal/signaltools.py

```diff
@@ -1439,10 +1439,14 @@ def wiener(im, mysize=None, noise=None):
     >>> import numpy as np
     >>> import matplotlib.pyplot as plt
     >>> from pylab import *
-    >>> x = np.random.random((40,40))         #Create a random image
-    >>> plot1 = plt.imshow(x)                 #Plot the image
-    >>> #Show image filtered with 5px by 5px window
-    >>> plot2 = plt.imshow(wiener(x, (5, 5)))
+    >>> img = np.random.random((40,40))    #Create a random image
+    >>> #Filter image with 5px by 5px window
+    >>> filtered_img = wiener(img, (5, 5))
+    >>> plt.figure()
```

Comment:
This creates a figure that you don't use. In the line below you get `f` as the figure anyways so creates a dummy output.